src/main/java/com/kakao/golajuma/auth/domain/service/ReadUserProfileService.java

```diff
@@ -0,0 +1,52 @@
+package com.kakao.golajuma.auth.domain.service;
+
+import com.kakao.golajuma.auth.infra.entity.UserEntity;
+import com.kakao.golajuma.auth.infra.repository.UserRepository;
+import com.kakao.golajuma.auth.web.dto.response.UserProfileResponse;
+import com.kakao.golajuma.vote.infra.repository.VoteRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class ReadUserProfileService {
+
+	private final UserRepository userRepository;
+	private final VoteRepository voteRepository;
+
+	public UserProfileResponse execute(Long userId) {
+
+		UserEntity userEntity = validateUserEntity(userId);
+		int createVoteCount = countCreatedVote(userId);
+		int participateVoteCount = countParticipatedVote(userId);
+
+		return userProfileConverter(userEntity, createVoteCount, participateVoteCount);
+	}
+
+	protected UserEntity validateUserEntity(Long userId) {
```

Comment:
validateUserEntity 라고 되어있어서 유효성 검사인 줄 알았는데 entity를 가져오는 역할을 하는 것 같아서 
```suggestion
	protected UserEntity getUserEntity(Long userId) {
```
이런식은 어떤가요

Reply:
넵 참고하도록 하겠습니다